### DISCUSSION TITLE
If 99% of the number of samples is less than 10ms, the percentiles value is confusing.

### DISCUSSION THREAD

**Discussion by huangmougou** [apache/skywalking#11823] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

In the 1-minute time window, 99% of the sampled values ​​are less than 10ms, a small part of which are 0ms, and most of which are greater than 0ms and less than 10ms. But the percentiles p55-p99 are all 0ms.
This #11063 is about the issue. But I didn't find a final solution to the problem. 

### What you expected to happen

When calculating percentiles, the default is to take 10ms as the bucket size. If the number of samples in the 0-10ms bucket exceeds 99%, the final value is obtained by multiplying the bucket index by 10ms, and the calculated value is 0ms.I personally think that the final calculated value can be the maximum value within the bucket range, or the median within the bucket range. rather than the minimum value within the bucket range.
![Snipaste_2024-01-23_18-36-22](https://github.com/apache/skywalking/assets/37232020/1ed776ae-41ac-4bc0-bd7e-1aa4c7953017)
![Snipaste_2024-01-23_18-37-06](https://github.com/apache/skywalking/assets/37232020/5c0096ef-cb7d-41e6-b1da-de3cf3dbdf94)

### How to reproduce

 The number of samples in the 0-10ms bucket exceeds 99%.

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
First, this is not a bug, but a discussion. Because codes are written in this way.

Then, OAL function provides the precision already, it could be defined by end users.
https://skywalking.apache.org/docs/main/next/en/concepts-and-designs/oal/#aggregation-function

There is no point to argue about using upper ceiling or lower. This would never end.

**Reply by huangmougou** [reply]
Thank you for your reply.
